descriptors/ModuleDescriptor-template.json

```diff
@@ -2,6 +2,21 @@
   "id": "${artifactId}-${version}",
   "name": "Inventory Module",
   "provides": [
+    {
+      "id": "inventory-batch",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/inventory/instancesCollection",
+          "permissionsRequired": ["inventory.instances.item.post"],
+          "modulePermissions": [
```

Comment:
Are these sufficient permissions? Do instances affect instance relationships, for example?